### PR TITLE
Node version 14 for CI

### PR DESCRIPTION
**Pull request by dartamonov-vertex**

### CODE REVIEW

**Comment by yoyo837** [comment]
CI failed.

**Comment by dartamonov-vertex** [comment]
Something must be off with CI. This PR failed with 0 changes to the source code.

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "build",
+      "scripts",
+      "acceptance-tests",
+      "webpack",
+      "jest",
+      "src/setupTests.ts",
+      "tslint:latest",
+      "tslint-config-prettier"
+    ]
+}
```
Comment (afc163) [inline tsconfig.json:24]:
```suggestion
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "declaration": true,
    "module": "esnext",
    "target": "esnext",
    "moduleResolution": "node",
    "jsx": "react",
    "skipLibCheck": true
  },
  "include": ["./src", "./typings/"],
  "typings": "./typings/index.d.ts",
  "exclude": [
    "node_modules",
    "build",
    "scripts",
    "acceptance-tests",
    "webpack",
    "jest",
    "src/setupTests.ts",
    "tslint:latest",
    "tslint-config-prettier"
  ]
}
```

File: src/Dropdown.tsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 import classNames from 'classnames';
-import { AnimationType, AlignType, BuildInPlacements, ActionType } from 'rc-trigger/lib/interface';
+import type {
+  AnimationType,
+  AlignType,
+  BuildInPlacements,
+  ActionType,
+} from 'rc-trigger/lib/interface';
 import Placements from './placements';
+import useAccessibility from './hooks/useAccessibility';
```
Comment (afc163) [inline src/Dropdown.tsx:12]: Does it belong to #182 which should not be commited here?

Comment (dartamonov-vertex) [reply src/Dropdown.tsx]: Updated with the push below

**Comment by dartamonov-vertex** [comment]
@afc163 @MadCcc This PR didn't add any new functional code. It was pretty much a test of the CI you introduced. If you don't want to use this PR, can you please create your own and test if it pass your CI.
You are welcome to use the fixes introduced in this PR if you need them:
* Added tsconfig.json to override father default config
* Fixed lint errors in src/Dropdown.tsx
* Updated node version to 14 in CI, because node 12 will be end of life'd in April

The last error on this PR - https://github.com/react-component/dropdown/pull/191/files#diff-c3c9c10a6e6fab8c821dc6ccbb2ffd57c5750035b6cfc76fed0afe78ef98121eR97 is related to the renamed variable. That line is not a new line. It just renames a variable to fix the lint issue.

**Comment by afc163** [comment]
图片

**Comment by MadCcc** [comment]
I will work on it.

**Comment by MadCcc** [comment]
updated https://github.com/react-component/dropdown/pull/192

**Pull request closed without merging**